<?xml version="1.0"?>
<launch>
    <group ns="stereo">
        <node name="left" pkg="usb_cam" type="usb_cam_node" output="screen">
            <param name="video_device" value="/dev/video1"/>
            <param name="image_width" value="640" />
            <param name="image_height" value="480" />
            <param name="pixel_format" value="yuyv" />
            <param name="camera_frame_id" value="left_camera"/>
            <param name="framerate" value="30"/>
            <param name="io_method" value="mmap"/>
            <param name="camera_name" value="left"/>
        </node>

        <node name="right" pkg="usb_cam" type="usb_cam_node" output="screen">
            <param name="video_device" value="/dev/video2"/>
            <param name="image_width" value="640" />
            <param name="image_height" value="480" />
            <param name="pixel_format" value="yuyv" />
            <param name="camera_frame_id" value="right_camera"/>
            <param name="framerate" value="30"/>
            <param name="io_method" value="mmap"/>
            <param name="camera_name" value="right"/>
        </node>

        <node name="stereo_image_proc" pkg="stereo_image_proc" type="stereo_image_proc">
            <param name="approximate_sync" value="true"/>
            <param name="queue_size" value="10"/>
        </node>
    </group>
</launch>
